platform/features/fixed/bundle.js

```diff
@@ -148,6 +140,82 @@ define([
                             {
                                 "items": [
                                     {
+                                        "property": "color",
+                                        "cssClass": "icon-T",
+                                        "title": "Text color",
+                                        "description": "Set text color",
+                                        "mandatory": true,
+                                        "control": "color"
+                                    },
+                                    {
+                                        "property": "size",
+                                        "title": "Text size",
+                                        "description": "Set text size",
+                                        "control": "select",
+                                        "options": [
```

Comment:
Could hoist out a TEXT_SIZES constant to the top (`var TEXT_SIZES = [8, 9, 10, 11, 12, 14, 16`...) and make options frm that

```
"options": TEXT_SIZES.map(function (sz) { 
    return { value: sz + "px", name: sz + " px" }; 
}),
```

Simplifies, removes opportunity for inconsistencies to emerge under maintenance

Reply:
using map is a great suggestion here.

would prefer to inline the array instead of defining a single use variable far away from where it is used.

Reply:
@psarram I'm refining the text sizes now, and will push to this branch shortly.

Reply:
@VWoeltjen Thanks for the feedback. I updated the code to use an array to make options.